lib/isMobilePhone.js

```diff
@@ -38,6 +38,7 @@ var phones = {
   'en-ZA': /^(\+?27|0)\d{9}$/,
   'en-ZM': /^(\+?26)?09[567]\d{7}$/,
   'es-ES': /^(\+?34)?(6\d{1}|7[1234])\d{7}$/,
+  'et-EE': /^(\+?372)?\s?(5|8[1-4])\s?([0-9]\s?){6,7}$/,
```

Comment:
You'll have to update [`src/lib/isMobilePhone.js`](https://github.com/chriso/validator.js/blob/master/src/lib/isMobilePhone.js#L29) too.